Delete timeout for mongodbatlas_private_endpoint resource too short

I keep hitting the 2 minute timeout when deleting `mongodbatlas_private_endpoint` resources, and it's playing havoc with my go tests:
```
TestAtlas 2020-02-24T10:05:19-05:00 command.go:158: Error: error deleting MongoDB Private Endpoints Connection(5e53dfd92a2f901f1d504107): timeout while waiting for state to become 'DELETED, FAILED' (last state: 'DELETING', timeout: 2m0s)
```
Can we get this timeout increased, or (even better) make it configurable in a `timeouts { ... }` block ?

--- a/mongodbatlas/resource_mongodbatlas_private_endpoint.go
+++ b/mongodbatlas/resource_mongodbatlas_private_endpoint.go
@@ -163,7 +163,7 @@ func resourceMongoDBAtlasPrivateEndpointDelete(d *schema.ResourceData, meta inte
 		Pending:    []string{"DELETING"},
 		Target:     []string{"DELETED", "FAILED"},
 		Refresh:    resourcePrivateEndpointRefreshFunc(conn, projectID, privateLinkID),
-		Timeout:    2 * time.Minute,
+		Timeout:    10 * time.Minute,
 		MinTimeout: 5 * time.Second,
 		Delay:      3 * time.Second,
 	}

